pages/embedded-portal.tsx

```diff
@@ -49,33 +52,28 @@ const EmbeddedPortal: NextPageWithLayout<Props> = ({
     setButtonText("Setting up Flatfile...");
   };
 
-  const flatfleSpace =
-    existingSpace?.flatfileData as unknown as FlatfileSpaceData;
+  const publishableKey = process.env.NEXT_PUBLIC_EMBEDDED_PUBLISHABLE_KEY;
+
+  if (!publishableKey) {
+    console.error("Missing NEXT_PUBLIC_EMBEDDED_PUBLISHABLE_KEY env var");
+    throw "Missing NEXT_PUBLIC_EMBEDDED_PUBLISHABLE_KEY env var";
+  }
+
   const [showSpace, setShowSpace] = useState(false);
   const spaceProps = {
-    accessToken: accessToken as string,
+    publishableKey,
     environmentId: environmentToken as string,
```

Comment:
should check further up the chain and not do `as string` here

Reply:
removed